crate/kmip/src/openssl/private_key.rs

```diff
@@ -154,6 +154,12 @@ pub fn kmip_private_key_to_openssl(private_key: &Object) -> Result<PKey<Private>
                     pad_be_bytes(&mut privkey_vec, 32);
                     PKey::private_key_from_raw_bytes(&privkey_vec, Id::ED25519)?
                 }
+                RecommendedCurve::CURVEED448 => {
+                    let mut privkey_vec = d.to_bytes_be();
+                    // 57 is privkey size on Ed448.
+                    pad_be_bytes(&mut privkey_vec, 57);
```

Comment:
we should use constant from kms_utils for all privkey sizes. Quite a few hardcoded values could be replaced.

Reply:
I strongly share this opininon. However, utils depends on kmip which makes it impossible to depend on utils.
I'm open to ideas :)